projects/reticulum/lib/ret_web/router.ex

```diff
@@ -21,7 +21,7 @@ defmodule RetWeb.Router do
   end
 
   scope "/api/login", RetWeb do
-    pipe_through :api
```

Comment:
I think it probably makes more sense to break this up into a few different pipelines and combine them together b/c you don't want `fetch_flash` to run on the API for example. For example create a `:shared` pipeline that has `fetch_session` and a `:dual` pipline that accepts both html and json.

Reply:
I don't think those can be separated- if we want to have flash messages from the auth flow via html (e.g. "successfully authenticated") then it has to be included for both